appinventor/components/src/com/google/appinventor/components/runtime/CSVFile.java

```diff
@@ -0,0 +1,306 @@
+package com.google.appinventor.components.runtime;
+
+import android.os.Environment;
```

Comment:
Write tests for new functionality

Reply:
+ more manual testing (especially of newly implemented reading workflow)

Reply:
Move on to tests after finishing #86

Reply:
To do:
- [ ] CSVFile tests
- [ ] ChartDataBase tests?
- [ ] CoordinateData tests?